### PR TITLE
Minor warnings fixes

### PR DESCRIPTION
**Pull request by wirednkod**

### CODE REVIEW

**Review by hbulgarini — CHANGES_REQUESTED** [review]
Please take into account that with this change, the placeholder identicon disappears when no account is selected  in the sender dropdown. You need to tweak this logic to do so:

https://github.com/paritytech/parity-bridges-ui/blob/master/src/components/AccountDisplay.tsx#L82

**Comment by hbulgarini** [comment]
I realized that 

> Please take into account that with this change, the placeholder identicon disappears when no account is selected in the sender dropdown. You need to tweak this logic to do so:
> 
> https://github.com/paritytech/parity-bridges-ui/blob/master/src/components/AccountDisplay.tsx#L82

Well this bug was introduced before this change but it worth to solve this issue in this PR if you agree.

**Comment by wirednkod** [comment]
I am (really hard) trying to understand how the change I made has ANYTHING to do with the placeholder disappearing. Can you please explain @hbulgarini ?

**Comment by hbulgarini** [comment]
> I am (really hard) trying to understand how the change I made has ANYTHING to do with the placeholder disappearing. Can you please explain @hbulgarini ?

the value 'init' was initially set so the check of the address existence does not hide the identicon. When we remove it , the identicon disappear as part of that check.

https://github.com/paritytech/parity-bridges-ui/blob/master/src/components/AccountDisplay.tsx#L82

File: src/components/AccountDisplay.tsx [hunk 1]
```diff
@@ -79,9 +81,9 @@ const AccountDisplay = ({ address = '', addressKind, balance, friendlyName, hide
 
   return (
     <Container onClick={onClick} className={classes.container}>
-      <div className={classes.icon}>{address && <AccountIdenticon address={address} />}</div>
-      <div className={classes.address}>
-        <p>{displayText()}</p>
+      <div className={classes.icon}>{<AccountIdenticon address={address} />}</div>
```
Comment (tomusdrw) [inline src/components/AccountDisplay.tsx:84]: I guess now it's not conditional we can actually remove the `'init'` stuff, no?

**Review by tomusdrw — APPROVED** [review]
lgtm!

File: src/components/AccountDisplay.tsx (hunk 1 above)
Comment (wirednkod) [reply src/components/AccountDisplay.tsx:84]: Yeap.. i did not? FFS!!!!

**Review by hbulgarini — APPROVED** [review]

**Pull request merged**